src/picknik_ur_gazebo_config/README.md

```diff
@@ -0,0 +1,7 @@
+# picknik_ur_gazebo_config
+
+A MoveIt Studio site configuration for UR5e simulated in Gazebo, based on the `picknik_ur_base_config` package available [here](https://github.com/PickNikRobotics/ur5e_sim_base_config).
```

Comment:
```suggestion
A MoveIt Studio site configuration for a UR5e simulated in Gazebo, based on the `picknik_ur_base_config` package available [here](https://github.com/PickNikRobotics/ur5e_sim_base_config).
```

Reply:
The URL is actually wrong too now that you highlighted this. Will fix.